confidant/routes/v1.py

```diff
@@ -392,6 +392,9 @@ def get_credential(id):
     cipher = CipherManager(data_key, cipher_version)
     _credential_pairs = cipher.decrypt(cred.credential_pairs)
     _credential_pairs = json.loads(_credential_pairs)
+
+    if authnz.user_is_user_type('user'):
+        logging.info(f"{authnz.get_logged_in_user()} fetching credential {id}")
```

Comment:
confidant is still python2 compat, so ideally use .format(), rather than f strings

Reply:
Thanks Ryan, good point.